environment/src/test/java/com/sequenceiq/environment/environment/flow/deletion/EnvDeleteActionsTest.java

```diff
@@ -174,22 +164,9 @@ void publicKeyDeleteActionTestHappyPath() {
                 EnvironmentStatus.PUBLICKEY_DELETE_IN_PROGRESS, ResourceEvent.ENVIRONMENT_PUBLICKEY_DELETION_STARTED);
     }
 
-    private void testFailure(Supplier<Action<?, ?>> deleteAction) {
```

Comment:
please rewrite test cases as well.

Reply:
Cannot rewrite these tests because the class is simplified.